samples/net/mqtt_publisher/README.rst

```diff
@@ -114,6 +114,27 @@ Open another terminal window and type:
 
 	$ mosquitto_sub -t sensors
 
+Building/Running On Linux
+==========================
+
+This sample can also be built and run on Linux.  The setup of the mosquitto
+server is the same, and the output should be similar to the Zephyr example.
+
+To build for a host POSIX OS:
+
+.. code-block:: console
+
+    $ make -f Makefile.posix
+
+To run:
+
+.. code-block:: console
+
+    $ sudo ./mqtt_publisher
+
+Note that sudo is necessary as the underlying mqtt library is using a
```

Comment:
delete the stray ``a`` in ``using a an async_sock``

Reply:
Done.